### PR TITLE
Reorganise things

### PR DESCRIPTION
**Pull request by iHiD**
This PR is an initial pass at getting the docs into a more final structure for launch.

### CODE REVIEW

File: contributing/configlet/generating-documents.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Configlet generating documents
 
-The secondary use of [configlet](./README.md) is generating documents.
+The secondary use of [configlet](../) is generating documents.
```
Comment (ErikSchierboom) [inline contributing/configlet/generating-documents.md]: How is this link rendered as it doesn't point to a document anymore.

File: contributing/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Contributing to Exercism
+
+This section of the docs looks at how to contribute to Exercism.
+It contains information specs for all various areas of Exercism as well as guides to things like becoming a maintainer and choosing a story for an exercise.
```
Comment (ErikSchierboom) [inline contributing/README.md]:
```suggestion
It contains information and specifications for all various areas of Exercism as well as guides to things like becoming a maintainer and choosing a story for an exercise.
```

File: contributing/tracks/concepts.md [hunk 3]
```diff
@@ -56,7 +56,7 @@ Here some examples of what could be covered.
 - Popular usages for a Concept
 - Common pitfalls in a Concept's use (e.g. failing to consider thread-safety)
 - Limitations on use that may catch out the unsuspecting developer
-- Alternative approaches addressed in other Concepts (e.g. the ••recursion** Concept might reference that the **Higher Order Functions** Concept offers an alternative approach to similar problems)
+- Alternative approaches addressed in other Concepts (e.g. the ••recursion** Concept might reference that the **Higher Order Functions\*\* Concept offers an alternative approach to similar problems)
```
Comment (ErikSchierboom) [inline contributing/tracks/concepts.md]:
```suggestion
- Alternative approaches addressed in other Concepts (e.g. the **recursion** Concept might reference that the **Higher Order Functions** Concept offers an alternative approach to similar problems)
```

File: contributing/configlet/generating-documents.md (hunk 1 above)
Comment (iHiD) [reply contributing/configlet/generating-documents.md]: Documents != URLs.

So `contibuting/configlet` will render a document (per the config.json) so this document having a link to `../` points to that node.

**Pull request merged**